{
  "id": "CVE-2022-40896",
  "references": [
    {
      "type": "exploit",
      "source": "NVD",
      "url": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages-part-2/",
      "time_stamp": "2023-07-26T21:04:00Z"
    }
  ]
}
